Source/WebCore/platform/graphics/texmap/TextureMapperLayer.cpp

```diff
@@ -80,6 +153,90 @@ TextureMapperLayer::~TextureMapperLayer()
     removeFromParent();
 }
 
+void TextureMapperLayer::processDescendantLayersFlatteningRequirements()
+{
+    // Traverse all children in depth first, post-order
+    for (auto* child : m_children) {
+        child->processDescendantLayersFlatteningRequirements();
+
+        if (child->flattensAsLeafOf3DSceneOr3DPerspective())
+            child->processFlatteningRequirements();
+    }
+}
+
+void TextureMapperLayer::processFlatteningRequirements()
+{
+    if (m_flattenedLayer)
```

Comment:
No need to null check, prio to restting to `nullptr`.

Reply:
Fixed